scripts/create-og-image.py

```
@@ -20,49 +23,52 @@ def create_og_image(input_path, output_path, target_width=1200, target_height=63
+            img.paste(logo_resized, (x_offset, y_offset))
         
         # Save the optimized image
-        img.save(output_path, 'PNG', optimize=True)
+        img.save(output_path, 'PNG', optimize=True, quality=95)
         print(f"✓ Created optimized OG image: {output_path}")
         print(f"  Dimensions: {img.size[0]}x{img.size[1]} (aspect ratio: {img.size[0]/img.size[1]:.2f}:1)")
+        print(f"  Logo size: {new_logo_width}x{new_logo_height} (scaled {scale:.2f}x)")
+        print(f"  Logo position: centered with {x_offset}px horizontal, {y_offset}px vertical padding")
         return True
         
     except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)